geopyspark/geopycontext.py

```diff
@@ -56,6 +56,8 @@ def __init__(self, pysc=None, **kwargs):
         self.sc = self.pysc._jsc.sc()
         self._jvm = self.pysc._gateway.jvm
 
+        self.pysc._gateway.start_callback_server()
```

Comment:
Could be done, but there may be value in just starting this server as a default action.  Might make it easy in the future for later applications that need to use the callback mechanism?

Maybe @echeipesh can weigh in on this?